examples/visualization/plot_eeglab_head_sphere.py

```diff
@@ -0,0 +1,133 @@
+
+rng = np.random.RandomState(0)
+data = rng.normal(size=(n_channels, 1)) * 1e-6
+fake_evoked = mne.EvokedArray(data, fake_info)
+fake_evoked.set_montage(biosemi_montage)
+
+###############################################################################
+# Calculate sphere origin and radius
+# ----------------------------------
+#
+# EEGLAB plots head outline at the level where head circumference is measured
+# in 10-20 system (a line going through Fpz, T8/T4, Oz and T7/T3 channels).
+# MNE places the head outline lower on the z dimension: approximattely at the
+# level of LPA, RPA fiducial markers. Therefore to use the EEGLAB layout we
+# have to move the origin of the reference sphere (a sphere that is used as a
+# reference when projecting channel locations to a 2d plane) a few centimeters
+# up.
+#
+# Instead of approximating this position by eye, as we did in the sensor
+# locations tutorial :ref:`tut-sensor-locations`, we will calculate it using
+# the position of Fpz, T8, Oz and T7 channels available in out montage.
+
+# first we obtain the 3d positions of selected channels (this should be easier
+# once we have ``.get_channel_positions()`` method available):
+check_ch = ['Oz', 'Fpz', 'T7', 'T8']
+ch_idx = [fake_evoked.ch_names.index(ch) for ch in check_ch]
+pos = np.stack([fake_evoked.info['chs'][idx]['loc'][:3] for idx in ch_idx])
+
+# now we calculate the radius from T7 and T8 x position
+# (we could use Oz and Fpz y positions as well)
+radius = np.abs(pos[[2, 3], 0]).mean()
+
+# then we obtain the x, y, z sphere center this way:
+# x - x position of the Oz channel (should be very close to 0)
+# y - y position of the T8 channel (should be very close to 0 too)
+# z - average z position of Oz, Fpz, T7 and T8 (their z position should be the
+#     the same, so we could also use just one of these channels), it should be
+#     positive and somewhere around `0.03` (3 cm)
+x = pos[0, 0]
+y = pos[-1, 1]
+z = pos[:, -1].mean()
+
+# lets print the values we got:
+print([f'{v:0.5f}' for v in [x, y, z, radius]])
+
+###############################################################################
+# Compare MNE and EEGLAB channel layout
+# -------------------------------------
+#
+# We already have the required x, y, z sphere center and its radius - we can
+# use these values passing them to the ``sphere`` argument of many
+# topo-plotting functions (by passing ``sphere=(x, y, z, radius)``).
+
+# create a two-panel figure with some space for the titles at the top
+fig, ax = plt.subplots(ncols=2, figsize=(8, 4), gridspec_kw=dict(top=0.9))
+
+# we plot the channel positions with default sphere - the mne way
+fake_evoked.plot_sensors(axes=ax[0], show=False)
+
+# in the second panel we plot the positions using the EEGLAB reference sphere
+fake_evoked.plot_sensors(sphere=(x, y, z, radius), axes=ax[1], show=False)
+
+# make x and limits the same in both panels
+ylm = ax[1].get_ylim()
+xlm = ax[1].get_xlim()
+ax[0].set_ylim(ylm)
+ax[0].set_xlim(xlm)
```

Comment:
passing `sharex` and `sharey` to `plt.subplots` above should make this unnecessary

Reply:
Oh, right, thanks!